### PR TITLE
Working basic login page, readme info with team

### PR DESCRIPTION
**Pull request by JC8747**
Changed Login title, added placeholder logo, updated contributor list… with names and github links, removed new login widget for rework.
**Team composition has changed so the readme team info will have to be updated again. **
Styling can be applied once a unified theme has been set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
...
+|                                                      [Jose Sustaita](https://github.com/JoseSustaita)                                                       |                                                       [Emilio Dellrodar](https://github.com/Dellrodar)                                                        |                                                      [Matthew Taylor](https://github.com/MTaylor-tech)                                                       |                                                       [Idong Essien](https://github.com/idongessien)                                                        |                                                      [Devin Ong](https://github.com/hoangvu71)
```
Comment (josesustaita) [inline README.md]: Since team composition has changed just update this to correctly reflect the team and it should be good to go!

**Review by josesustaita — DISMISSED** [review]
Everything looks great, just change the readme to correctly reflect the team!

**Review by Cireimu — CHANGES_REQUESTED** [review]
Looks good, just need you to update the team composition as well as either get rid of all the place holder images or replace them with all five team member's GitHub profile pics.

I will also need you to add a .env.sample file with the correct variables so that the CI/CD is happy and no longer blocks merge.

**Review by Cireimu — APPROVED** [review]

**Review by josesustaita — APPROVED** [review]
Thank you for updating the readme profile. Look good!

**Pull request merged**